eng/versioning/version_client.txt

```diff
@@ -7,9 +7,9 @@ com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-anomalydetector;3.0.0-beta.1;3.0.0-beta.2
 com.azure:azure-ai-formrecognizer;3.0.0;3.1.0-beta.1
 com.azure:azure-ai-textanalytics;5.0.0;5.1.0-beta.1
-com.azure:azure-core;1.8.0;1.8.1
+com.azure:azure-core;1.8.1;1.9.0-beta.1
```

Comment:
Should we remove the `unreleased` tags too for core?

Reply:
Yeah, after all releases are added into the PR I'll go through and remove all the unreleased Core dependencies.